linalg/vector.hpp

```diff
@@ -86,6 +86,11 @@ class Vector
    Vector(int size_, MemoryType mt)
       : data(size_, mt), size(size_) { }
 
+   /// Create a vector using a braced initializer list
+   template <int N>
+   explicit Vector(const double (&values)[N]) : Vector(N)
+   {std::copy(values, values + N, GetData());}
```

Comment:
```suggestion
   { std::copy(values, values + N, GetData()); }
```
Minor style…

Reply:
Ah, I wasn't aware that `astyle` didn't cover things like this - thanks!